android/source/VideoLocker/src/org/edx/mobile/util/DateUtil.java

```diff
@@ -86,4 +86,21 @@ public static String getModificationDate() {
         }
     }
 
+    /**
+     *  This function returns course start date in the MMMM dd, yyyy format
+     */
+    public static String formatCourseNotStartedDate(String date) {
```

Comment:
@rohan-dhamal-clarice - I have added the test case for the above function. Please verify.